javacutil/src/main/java/org/checkerframework/javacutil/AnnotationBuilder.java

```diff
@@ -147,6 +150,62 @@ public static AnnotationMirror fromClass(
         return res;
     }
 
+    /**
+     * Creates an {@link AnnotationMirror} given by a particular annotation class. Each element is
+     * set to its default value. Errs if any element does not have a default value.
+     *
+     * @param elements the element utilities to use
+     * @param aClass the annotation class
+     * @return an {@link AnnotationMirror} of type given type
+     */
+    public static AnnotationMirror fromClassWithDefaults(
+            Elements elements, Class<? extends Annotation> aClass) {
+        AnnotationMirror result = fromName(elements, aClass.getCanonicalName());
```

Comment:
This should either use `fromClass`, which checks for `null`, or check for `null` here.